Library/Homebrew/extend/ARGV.rb

```diff
@@ -25,7 +25,7 @@ def formulae
   def resolved_formulae
     require "formula"
     @resolved_formulae ||= (downcased_unique_named - casks).map do |name|
-      if name.include?("/")
+      if name.include?("/") || File.exist?(name)
```

Comment:
Thanks for the explanation! It's going to be interesting (though very unlikely) once there's a file named `foo.rb` and a formula named `foo.rb` (perfectly legal formula name in file `foo.rb.rb`). I have no idea what this will resolve to in this case. 😀

> You are right we probably should check file ext.

:+1: